addons/binding/org.openhab.binding.opendaikin/src/main/java/org/openhab/binding/opendaikin/internal/api/SensorInfo.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.opendaikin.internal.api;
+
+import java.util.Optional;
+
+import org.openhab.binding.opendaikin.handler.OpenDaikinAcUnitHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Holds information from the get_sensor_info call.
+ *
+ * @author Tim Waterhouse - Initial contribution
+ *
+ */
+public class SensorInfo {
+    private static Logger logger = LoggerFactory.getLogger(OpenDaikinAcUnitHandler.class);
+
+    public Optional<Double> indoortemp;
+    public Optional<Double> indoorhumidity;
+    public Optional<Double> outdoortemp;
+
+    private SensorInfo() {
+    }
+
+    public static SensorInfo parse(String response) {
+        logger.debug("Parsing {}", response);
+        SensorInfo info = new SensorInfo();
+
+        for (String keyValuePair : response.split(",")) {
+            if (keyValuePair.contains("=")) {
```

Comment:
If this would fail the fields are not initialized with `Optional.empty()` and will cause problems in the rest of the code. So better initialize the fields above (and then you can omit the else cases below) to avoid this problem.